dotCMS/src/integration-test/java/com/dotcms/auth/providers/jwt/JsonWebTokenUtilsIntegrationTest.java

```diff
@@ -102,10 +110,13 @@ public void get_user_in_token_modified()
                 JsonWebTokenFactory.getInstance().getJsonWebTokenService();
         assertNotNull(jsonWebTokenService);
 
+        final String jwtId = APILocator.getUserAPI().loadUserById(userId).getRememberMeToken();
         //Generate a new token
-        String jsonWebToken = jsonWebTokenService.generateUserToken(new UserToken(jwtId,
-                userId, date, DateUtil.daysToMillis(2)
-        ));
+        final UserToken userToken = new UserToken.Builder().id(jwtId).subject(userId).modificationDate(date)
+                .expiresDate(DateUtil.daysToMillis(2))
+                .build();
+        String jsonWebToken = jsonWebTokenService.generateUserToken(userToken);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'jsonWebToken' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5164094)